AntWars/AI.cs

```diff
@@ -0,0 +1,75 @@
+﻿using AntWars.AI;
```

Comment:
Naja, sogesehen ist diese klasse ja extern und sollte somit nicht in das klassendiagramm